contracts/lib/SushiswapV2Library.sol

```diff
@@ -0,0 +1,85 @@
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0;
+
+import '@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol';
+
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+
+library SushiswapV2Library {
```

Comment:
Instead of pulling in the entire file, you could just create an interface for the SushiswapV2Library and UniswapV2Library. We typically put all the actual contract files in the `external` folder